tox.ini

```diff
@@ -47,11 +47,14 @@ deps =
 
 commands =
     "{toxinidir}/.travis/environment"
-    coverage run -p "{envdir}/bin/trial" {posargs:klein}
+
+    trial: "{envdir}/bin/trial" {posargs:klein}
```

Comment:
There is an `{envbindir}` variable this scenario. No need to change this, though.